waku/README.md

```diff
@@ -0,0 +1,287 @@
+# `waku`
+
+## Table of contents
+
+- [What is Waku?](#what-is-waku)
+- [Waku versioning](#waku-versioning)
+- [What does this package do?](#what-does-this-package-do)
+- [Waku package files](#waku-package-files)
+
+## What is Waku?
+
+Waku is a communication protocol for sending messages between Dapps. Waku is a fork of the [Ethereum Whisper subprotocol](https://github.com/ethereum/wiki/wiki/Whisper), although not directly compatible with Whisper, both Waku and Whisper subprotocols can communicate [via bridging](https://github.com/vacp2p/specs/blob/master/specs/waku/waku-1.md#backwards-compatibility).
+
+Waku was [created to solve scaling issues with Whisper](https://discuss.status.im/t/fixing-whisper-for-great-profit/1419) and [currently diverges](https://github.com/vacp2p/specs/blob/master/specs/waku/waku-1.md#differences-between-shh6-and-waku1) from Whisper in the following ways:
+
+- RLPx subprotocol is changed from `shh/6` to `waku/1`.
+- Light node capability is added.
+- Optional rate limiting is added.
+- Status packet has following additional parameters: light-node, confirmations-enabled and rate-limits
+- Mail Server and Mail Client functionality is now part of the specification.

```

Comment:
Worst naming ever, but I guess I will never be able to push through the renaming of this to `history node`.

Reply:
He who dares wins. I support this change and would lobby for its implementation.

If we can agree on the better name. I would propose `persistence node`, or `echo node`.

One thing I am 100% in agreement with, the Mailserver is a stupid name.

Reply:
Yeah, makes you think of technology from 1970s, not something developed in 2020s.

Reply:
@jakubgs this may interest you https://discuss.status.im/t/wherefore-art-thou-mailserver-treatise-on-waku-terminology/1664